index.js

```diff
@@ -95,7 +113,7 @@ function MinkeLite(config) {
 
   this.config.dir_path = this.config.dir_path || "./"
   this.config.db_name = this.config.in_memory ? ":memory:" : (
-    this.config.db_name || "minkelite.db" );
+    this.config.db_name || '' );
```

Comment:
https://www.sqlite.org/tempfiles.html discussed temp file usage in sqlite, but it's not clear what happens if node exits without closing the connection.  Wrote a test code for these 3 cases:

`var params = [
	{testId: 'In-Memory', data: {'in_memory': true, 'db_name': 'minkelite_test.db', 'overwrite': true}},
	{testId: 'On-File/Named', data: {'in_memory': false, 'db_name': 'minkelite_test.db', 'overwrite': true}},
	{testId: 'On-File/Anonymous', data: {'in_memory': false, 'db_name': '', 'overwrite': true}}
];`